test/remote-replication.test.js

```diff
@@ -0,0 +1,90 @@
+describe('Remote Replication', function() {
+  beforeEach(function(done) {
+    var test = this;
+
+    test.serverApp = loopback();
+    test.serverApp.use(function(req, res, next) {
+      console.log(req.method, req.url);
+      next();
+    });
+    test.serverApp.use(loopback.rest());
+    test.serverApp.enableAuth();
+    test.clientApp = loopback();
+    this.serverApp.set('legacyExplorer', false);
+    var settings = {
+      base: 'PersistedModel',
+      trackChanges: true,
+      dataSource: null,
+      http: {
+        path: 'my-model'
+      }
```

Comment:
IIRC this setting is not honoured by the remoting connector, see https://github.com/strongloop/loopback-connector-remote/issues/16. Perhaps I am wrong?